### ISSUE TITLE
File to import not found or unreadable: neat.

### ISSUE THREAD

**Issue by gnumarcelo** [thoughtbot/neat#546]
I'm trying to install and import `neat` into my `rails 4.2.6` app but it doesn't work. I've followed the steps below:

1 - add `gem  'bourbon' and gem  'neat'` to my Gemfile;
2 - Run `bundle install`; --> this installed bourbon (4.3.2) and neat (2.0.0)
3 - import it into my public.scss file:
```
@import "bourbon";
@import "neat";
```

But as soon as I restart my server an open my browser I get:
`File to import not found or unreadable: neat.`

I'm using RVM and a little piece of my Gemfile:

```
source 'https://rubygems.org'

ruby '2.3.3'

gem 'rails', '4.2.6'
gem 'mongoid', '~> 5.1.0'
gem 'mongoid-enum'
gem 'bson_ext'
gem 'sass-rails', '~> 5.0'
gem 'bootstrap-sass'
gem 'bourbon'
gem 'neat'
...
```

**Comment by whmii** [comment]
I just did an init of a new rails app and was able to get everything working. I also tried out a [suspenders](https://github.com/thoughtbot/suspenders) install which also worked.

Not sure what the issue is. I noticed you had named your Sass file `public.scss` which makes me wonder if it is in the `/public` directory. I believe doing this limits it's access to the established vars of the asset pipleine.

I dont think it is a problem with Neat. Let me know if you were able to resolve the problem.

**Comment by kgcreative** [comment]
It may be that you may need to add `/bourbon-neat/core` to your application's sass include paths.

ex:
```
module.exports = {
	sass: {
		mode: 'native',
		options: {
			includePaths: [
				'<%= pkg._npmpath %>/bourbon/core',
				'<%= pkg._npmpath %>/bourbon-neat/core',
				'<%= pkg._npmpath %>/normalize.css/',
				'<%= pkg._npmpath %>/font-awesome/scss/'
			],
      precision: 10,
			outputStyle: 'nested',
			sourceMap  : true
		},
		files  : {
      '<%= pkg._themepath %>/css/master-temp.css'             : '<%= pkg._themepath %>/scss/master.scss'
		}
	}

};
```

**Comment by bjacobs09** [comment]
I rolled back from neat 2.0 to 1.8 and it worked

**Comment by gretzky** [comment]
Can we get an actual fix on this? Neat 2.0 is too different from Neat 1.8 to do rollbacks when you've already migrated to 2.0...
